sys-apps/ucspi-ssl/ucspi-ssl-0.10.11.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit fixheadtails qmail
+
+DESCRIPTION="Command-line tools for building SSL client-server applications"
+HOMEPAGE="https://www.fehcom.de/ipnet/ucspi-ssl.html"
+SRC_URI="https://www.fehcom.de/ipnet/ucspi-ssl/${P}.tgz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
+IUSE="bindist perl libressl"
+
+DEPEND="<sys-libs/fehqlibs-13:=
```

Comment:
Your introduced fehqlibs doesn't seem to provide ABI soname slot, so `:=` is useless here. Do you know if upstream libs change ABI often?

Reply:
> Do you know if upstream libs change ABI often?

No idea.